### ISSUE TITLE
Jest test breaks when Highcharts uses highchartsAccessibility

### ISSUE THREAD

**Issue by Florian8** [highcharts/highcharts#20385]
#### Expected behavior
The jest test checks that the right value is shown in a progress circle and passes.

#### Actual behavior
The rendering of the component in the jest test fails. It says  _TypeError: (0 , accessibility_1.default) is not a function._ 
If I don't use highchartsAccessibility the test passes. 
As a result we have to mock all Highcharts graphs in our module test because the Highcharts graph breaks the entire test.

#### Product version
"highcharts": "11.1.0",
"highcharts-react-official": "3.2.1",
"jest": "29.7.0",
 "react-test-renderer": "16.14.0",

**Comment by uanderso** [comment]
We have the same error. Any progress with this bug?

**Comment by karolkolodziej** [comment]
@uanderso could you please share your case? How can we recreate that?

**Comment by uanderso** [comment]
> @uanderso could you please share your case? How can we recreate that?

The HighChart PieChart-component is used inside a React component using Emotion (which we are removing soon).

We have now mocked the Accessibility module in our tests as some tests causes a timeout in our pipeline if not mocked.
This is how we mocked the component if some one else has the same problem.
jest.mock("highcharts/modules/accessibility", () => ({
    __esModule: true,
    default: () => {
        return "";
    },
    setOptions: () => jest.fn(),
}));

jest.mock("highcharts-react-official", () => ({
    __esModule: true,
    default: () => {
        return "";
    },
    setOptions: () => jest.fn(),
}));
